source/install/self-managed-changelog.md

```diff
@@ -5,11 +5,126 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v6.0.
 
 Latest Mattermost Releases:
+- [Release v7.5 - Feature Release](#release-v7-5-feature-release)
 - [Release v7.4 - Feature Release](#release-v7-4-feature-release)
 - [Release v7.3 - Feature Release](#release-v7-3-feature-release)
 - [Release v7.2 - Feature Release](#release-v7-2-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1-extended-support-release)
-- [Release v7.0 - Major Release](#release-v7-0-major-release)
+
+## Release v7.5 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**v7.5.0 release day: 2022-11-16**
+
+### Important Upgrade Notes
+
+ - Adds a new schema migration to ensure ``ParentId`` column is dropped from the ``Posts`` table. Depending on the table size, if the column is not dropped before, a significant spike in database CPU usage is expected on MySQL databases. Writes to the table will be limited during the migration.
```

Comment:
"if the column is not dropped before" -- before what? Is "before" needed? Would the sentence be correct if "before" was removed?

Reply:
@calebroseland Please help confirm.

Reply:
I think this one may have been from @isacikgoz ?

Reply:
Correct, pinged a wrong person accidentally. @isacikgoz please help confirm the above question from Carrie.